### PR TITLE
Bid selection 2.0

### PR DESCRIPTION
**Pull request by lepistone**
depends on:
- [x] OCA/purchase-workflow#109

### CODE REVIEW

**Comment by gurneyalex** [comment]
@lepistone just to check that this will work once https://github.com/OCA/purchase-workflow/pull/109 is merged, I altered the travis config.

**Comment by gurneyalex** [comment]
still :red_circle:

**Comment by lepistone** [comment]
thanks @gurneyalex, I'm investigating

**Comment by lepistone** [comment]
@gurneyalex it's green now. Thanks for bringing this up.

**Comment by gurneyalex** [comment]
@lepistone  red on runbot:

```
loading ngo_purchase_requisition/view/purchase_order.xml
openerp.addons.base.ir.ir_ui_view: Field `incoterm_id` does not exist
```

**Comment by lepistone** [comment]
@gurneyalex maybe your hack to pull the other PR is for travis only?

**Comment by gurneyalex** [comment]
@lepistone: true (sorry I forgot about this...)

**Comment by gurneyalex** [comment]
:+1:

**Comment by yvaucher** [comment]
@lepistone dependancy merged

**Comment by gurneyalex** [comment]
amended head to get a rebuild

**Comment by yvaucher** [comment]
:+1:

File: ngo_purchase_requisition/view/purchase_requisition.xml [hunk 1]
```diff
@@ -114,18 +115,22 @@
...
-                    <field name="location_id" groups="stock.group_locations"/>
                     <field name="minimum_planned_date" invisible="1"/>
-                    <field name="origin"/>
                     <field name="state"/>
                     <!-- purchase_requisition_bid_selection removed button purchase_cancel -->
                     <!-- purchase_requisition_bid_selection removed button purchase_confirm -->
                     <!-- purchase_requisition_bid_selection removed button purchase_approve -->
                     <!-- purchase_requisition_bid_selection change button icon -->
+                    <field name="bid_selection_remark"/>
+                    <button name="%(purchase_requisition_bid_selection.action_update_bid_selection_remark)d" type="action" string="Update selection remark" icon="gtk-ok"/>
```
Comment (gurneyalex) [inline ngo_purchase_requisition/view/purchase_requisition.xml:125]: needs fixing: this is being changed in https://github.com/OCA/purchase-workflow/commit/7752251f32ffe0b7e2779f61eacba574abcc1702#diff-1c73f19583b95c25255f4c412232d2b5R21 (https://github.com/OCA/purchase-workflow/pull/111)

**Pull request merged**